charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -21,4 +21,4 @@ maintainers:
 name: rancher-vsphere-csi
 sources:
 - https://github.com/kubernetes-sigs/vsphere-csi-driver
-version: 3.3.0-rancher2
+version: 3.3.0-rancher3
```

Comment:
I notice the image tag for `rancher/mirrored-cloud-provider-vsphere-csi-release-driver` and `rancher/mirrored-cloud-provider-vsphere-csi-release-syncer` are bumped to  v3.3.1, 
Could you bump the version and appVersion to `3.3.1-rancher1`? 

Side note: there are no changes except image tags in the file `vsphere-csi-driver.yaml` between the upstream release 3.3.1 and 3.3.0 (l[ink](https://github.com/kubernetes-sigs/vsphere-csi-driver/compare/v3.3.0...v3.3.1)), which is good.